ichnaea/locate/provider.py

```diff
@@ -550,6 +554,12 @@ class FallbackProvider(Provider):
     location_type = Position
     source = DataSource.Fallback
 
+    def __init__(self, settings, *args, **kwargs):
+        self.url = settings['url']
+        self.ratelimit = int(settings['ratelimit'])
```

Comment:
can you change that to `self.ratelimit = int(settings.get('ratelimit', '0'))`, so the config is optional?

Feel free to make that into something better that uses `None` as a not-set key as well, I was too lazy ;)